src/main/java/worldofzuul/Rooms/Property.java

```diff
@@ -0,0 +1,39 @@
+package worldofzuul.Rooms;
+
+import worldofzuul.EnergySources.EnergySource;
```

Comment:
You could also use `import worldofzuul.EnergySources.*;`, but it doesn't matter when it is only from a single file.